xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -325,11 +328,11 @@ void CGUIDialogPVRTimerSettings::InitializeSettings()
   AddTypeDependentEnableCondition(setting, SETTING_TMR_NEW_EPISODES);
 
   // Pre and post record time
-  setting = AddSpinner(group, SETTING_TMR_BEGIN_PRE, 813, 0, m_iMarginStart, 0, 1, 60, 14044);
+  setting = AddSpinner(group, SETTING_TMR_BEGIN_PRE, 813, 0, m_iMarginStart, 0, MARGIN_SPINNER_STEP, MARGIN_SPINNER_MAX, 14044);
```

Comment:
To allow this to add a 'current' setting I believe you would need to change the AddSpinner line for pre and post record time to use this format:
```setting = AddSpinner(group, SETTING_TMR_BEGIN_PRE, 813, 0,m_iMarginStart , StartTimeFiller);```
and
```setting = AddSpinner(group, SETTING_TMR_END_POST, 813, 0,m_iMarginEnd , EndTimeFiller);```

And then create ```EndTimeFiller``` and ```StartTimeFiller``` functions similar to ```DaysFiller``` (https://github.com/xbmc/xbmc/blob/master/xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp#L837)

(Maybe it could be one function which distinguishes between the settings like DaysFiller does)

Hope this helps.